2. String/binary_add/binary_addition_chalti.cpp

```diff
@@ -11,32 +11,31 @@
 #include <bits/stdc++.h>
 using namespace std;
 
-void solve(string x, string y) { 
-    string result = "";  //initialize the resultant string as null
-    int sumOfdigit = 0;  //initialize the sum of the digit as zero 
-
-    //traverse both of the strings from last digit to first digit
-    while (x.size() >= 0 || y.size() >= 0 || sumOfdigit == 1) 
-    { 
-        sumOfdigit += ((x.size() >= 0)? x[x.size()] - '0': 0); 
-        sumOfdigit += ((y.size() >= 0)? y[y.size()] - '0': 0); 
-        result = char(sumOfdigit % 2 + '0') + result; //if current result if 1 or 0 then add it with the resultant string
-        sumOfdigit /= 2;  //carry
-        x.size()--; y.size()--;  //increase the digit number or move to the next digit
-    } 
-    cout<<result<<endl; //print the ans
-}
-
-int main() {
+void solve() { 
     int t;
     cin>>t;
     while(t-->0){

```

Comment:
take the test case in the main function and rest of the code in solve function. please refer to the template in Contributing.md